life_expectancy/cleaning.py

```diff
@@ -2,46 +2,85 @@
 Script to load, clean and save data
 """
 import argparse
+import pathlib
 import pandas as pd
 
 
 
-# life_path = pathlib.Path.cwd()
-# data_path = life_path.joinpath('data')
+PARENT_PATH = pathlib.Path(__file__).parent
+FILE_PATH = PARENT_PATH / 'data'
+INPUT_FILE_NAME = 'eu_life_expectancy_raw.tsv'
+OUTPUT_FILE_NAME = 'pt_life_expectancy.csv'
+INPUT_FILE_PATH = FILE_PATH / INPUT_FILE_NAME
+OUTPUT_FILE_PATH = FILE_PATH / OUTPUT_FILE_NAME
```

Comment:
Yes, yes, yes! The beauty of doing this is that then it become easy to abstract file configurations to external files if we need to! Just like how Lupa does it.